content/docs/azure/azureEndtoEnd.md

```diff
@@ -0,0 +1,171 @@
++++
+title = "End-to-End Pipeline Deployment on Azure"
+description = "An end-to-end guide to using the azurepipeline example"
+weight = 50
++++
+## Introductions
+### Overview of Azure and AKS
+
+Microsoft Azure is an open, flexible, enterprise-grade cloud computing platform running on Microsoft infrastructure. The platform has various services, including (SERVICES). 
+
+The [Azure CLI]((https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)) is a set of tools that you can use to interact with Azure from the command line.
+
+Azure Kubernetes Service (AKS) on Azure allows you to deploy containerized applications, within which you describe the resources your application needs, and AKS will manage the underlying resources automatically. This workflow is especially efficient at scale.
+### The overall workflow
+This guide will take you through using your Kubeflow deployment to build a Machine Learning pipeline on Azure. This guide uses a sample pipeline to detail the process of creating an ML workflow from scratch. You will create and run a pipeline that processes data, trains a model, and then registers and deploys that model as a webservice. 
+
+ To build your pipeline, you must create and build containers using Docker images. The containers are used to abstract the dependencies for each step of the pipeline. The containers can be managed using [Azure's portal](https://ms.portal.azure.com/#home), specifically using the Container Registry to store the containers in the cloud. The containers are then pulled by Kubeflow as they are needed in each step of the pipeline.
+
+By following this guide, you will be able to:
+
+- Set up Kubeflow in an AKS Cluster
+- Create and compile a pipeline that can:
+	- Preprocess data
+	- Train a model
+	- Register the model to ACR ([Azure Container Registry](https://docs.microsoft.com/en-us/azure/devops/pipelines/languages/acr-template?view=azure-devops))
+	- Profile the model to optimize compute resources in AML (Azure Machine Learning)
+	- Deploy the model to AML
+- Interact with and customize your deployment
+- Test and use your deployed model
+
+When your pipeline is finished, you should be able to see a registered image, model, and deployment in your Azure ML workspace. You will then be able to visit the scoring URI and upload images for scoring in real time.
```

Comment:
s/is finished/has finished running

(Active voice, to indicate the pipeline must finish doing something, rather than the reader having to finish the pipeline.)